core/src/scss/components/hero/_hero.scss

```diff
@@ -40,6 +41,31 @@ $_su-hero-height: (
   @include grid-media('xl') {
     min-height: map-get($_su-hero-height, 'xl');
   }
+
+  // The content
+  .su-hero__card {
+    background: $su-color-white;
+    position: relative;
+    z-index: 10;
+
+    @include grid-media('md') {
+      @include margin(45px null);
+      left: 45px;
+      max-width: 400px;
+    }
+
+    @include grid-media('lg') {
+      max-width: 450px;
+    }
+
+    @include grid-media('xl') {
+      @include margin(96px null 0 null);
+      bottom: 48px;
+      left: 48px;
+      max-width: 450px;
+      top: auto;
```

Comment:
In the future you might want to finetune the position of the card when the banner is full browser width, but that's probably for another PR. For our WordPress template, if the banner is full width, we line up the edge of the box with the edge of the center container, e.g., if the box/card is on the left, its left edge lines up with the left edge of the logo lockup.

Reply:
FYI this is what we have

![Screen Shot 2019-10-23 at 5 30 44 PM](https://user-images.githubusercontent.com/42749717/67443743-f8e52600-f5ba-11e9-85b4-116d5cb15c98.png)

Reply:
This is existing code that was outside its scope. Before, this `.su-card` class was not nested under the `.su-hero` class and was effecting cards elsewhere. What you have in the redwood theme looks great.